<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapTrackItem</class>
 <widget class="QWidget" name="MapTrackItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px solid rgb(180,180,180)</string>
  </property>
  <widget class="QLabel" name="m_labelImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(180,180,180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelDateTime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>2017-05-26 14:25:25</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelPosition">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>新疆博览馆</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
